Example/AddressGeocoderExample/ViewController.swift

```diff
@@ -26,13 +26,15 @@ class ViewController: UIViewController, Geocodable {
             return
         }
 
-        geocodeLocality(zipCode,
-                        forCountry: "US",
-                        completion: {(result: Result<GeocodedAddress, GeocodingError>) -> () in
-                            if let address = result.value {
-                                self.addressLabel.text = address.locality + ", " + address.state
-                            }
-        })
+        geocodeAddress(
+            from: zipCode,
```

Comment:
the geocode address with `from` sounds a little bit weird to me when we pass just a zip code. What do you think about `using` instead?